### PR TITLE
#1075 Multi Organization Support

### PR DESCRIPTION
**Pull request by ahay-agile6**
This PR adds  Multi Organization Support
for `DHPrimeAdmins` group on the frontend

## Changes
frontend:
- add dropdown wrapper to header
- add logic to add dropdown switch when user is part of `DHPrimeAdmins` group
- switch organization on dropdown value change
- fetch reports on organization dropdown value change
- fetch organization information on organization dropdown value change
- save organization in `sessionStorage` for subsequent requests

backend:
- allow `checkAuthenticated` to receive "organization" header
- ensure user is authorized to view reports based on auth service claims
- fetch reports based on organization sent in header
- if no organization is sent in the header, previous workflow of retrieving user's organization works

### Testing
- [ ] Tested locally?
- [ ] Ran `quickTest all`?
- [ ] Ran `./prime test` against local Docker ReportStream container?
- [ ] Downloaded a file from `http://localhost:7071/api/download`?
- [ ] Added tests?

### Security
- [ ] Did you check for sensitive data, and remove any? 
- [ ] Does logging contain sensitive data?  
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

### Process
- [ ] Includes a summary of what a code reviewer should verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Security
*List potential security threats and mitigations in the PR*

## Fixes
*List GitHub issues this PR fixes*
- #1075 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
n/a

### CODE REVIEW

**Review by MauriceReeves-usds — APPROVED** [review]
Looks good to me, but I'm going to defer to @mry-usds and @drew-usds for final approval.

File: frontend/src/js/prime-web-receiver-pre.js [hunk 1]
```diff
@@ -1,14 +1,44 @@
+/** Convert Org name
```
Comment (mry-usds) [inline frontend/src/js/prime-web-receiver-pre.js:1]: Nice - putting these in separate/distinct funcs

**Review by mry-usds — APPROVED** [review]
Great work - tested it out -- works like a charm!

**Comment by mry-usds** [comment]
The SonarCloud analysis may be due to doing this in your own repo (we typically branch in the prime-data-hub repo) - you might have to merge in any changes to master first...

**Pull request merged**